.devcontainer/Dockerfile

```diff
@@ -5,12 +5,18 @@
 ARG VARIANT="6.0-focal"
 FROM mcr.microsoft.com/vscode/devcontainers/dotnet:0-${VARIANT}
 
-# Set up machine requirements to build the repo
+# Setup the gh (GitHub) CLI signing key.
+RUN curl -fsSL -o /usr/share/keyrings/githubcli-archive-keyring.gpg https://cli.github.com/packages/githubcli-archive-keyring.gpg && \
+  echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages $(lsb_release -cs) main" | \
+  tee /etc/apt/sources.list.d/github-cli2.list > /dev/null
```

Comment:
why the 2 in github-cli2.list?

Reply:
The name of the file doesn't really matter (see `man sources.list`). The two is because this is version 2 of the CLI.